README.md

```diff
@@ -0,0 +1,262 @@
+    comments. Feel free to reach out to the RFC assignee in particular to get
+    help identifying stakeholders and obstacles.
+  - The team will discuss the RFC pull request, as much as possible in the
+    comment thread of the pull request itself. Other discussion (e.g. on
+    Discord) will be summarized on the pull request comment thread.
+  - RFCs rarely go through this process unchanged, especially as alternatives
+    and drawbacks are shown. You can make edits, big and small, to the RFC to
+    clarify or change the design, but make changes as new commits to the pull
+    request, and leave a comment on the pull request explaining your changes.
+    **Specifically, do not squash or rebase commits after they are visible on the
+    pull request.**
+  - At some point, a member of the team will propose a "motion for final
+    comment period" (FCP), along with a *disposition* for the RFC (merge, close,
+    or postpone).
+    - This step is taken when enough of the tradeoffs have been discussed that
+      the team is in a position to make a decision. That does not require
+      consensus amongst all participants in the RFC thread (which is usually
+      impossible). However, the argument supporting the disposition on the RFC
+      needs to have already been clearly articulated, and there should not be a
+      strong consensus *against* that position. Team members use their best
+      judgment in taking this step, and the FCP itself ensures there is ample
+      time and notification for stakeholders to push back if it is made
+      prematurely.
+    - For RFCs with lengthy discussion, the motion to FCP is usually preceded by
+      a *summary comment* trying to lay out the current state of the discussion
+      and major tradeoffs/points of disagreement.
+    - Before actually entering FCP, *all* team members must sign off; this
+      might be the point at which many team members first review the RFC in full
+      depth.
+  - The FCP lasts ten calendar days, so that it is open for at least 5 business
+    days. It is also advertised widely,
+    e.g. in [This Week in Nushell](https://www.notion.so/This-Week-in-Nu-ed589e19f1be47fa9ef562c1757e7ba1).
+    This way all stakeholders have a chance to lodge any final objections
+    before a decision is reached.
+  - In most cases, the FCP period is quiet, and the RFC is either merged or
+    closed. However, sometimes substantial new arguments or ideas are raised,
+    the FCP is canceled, and the RFC goes back into development mode.
+
+## The RFC life-cycle
+[The RFC life-cycle]: #the-rfc-life-cycle
+
+Once an RFC becomes "active" then authors may implement it and submit the
+feature as a pull request to the Nushell repo. Being "active" is not a rubber
+stamp, and in particular still does not mean the feature will ultimately be
+merged; it does mean that in principle all the major stakeholders have agreed
+to the feature and are amenable to merging it.
```

Comment:
Something else that I noticed in the Rust RFC process is that people would resist an RFC getting accepted, even though its acceptance didn't mean that it would become part of the project. Rather, it just meant it could be implemented.

Going along with the idea above we might want to call out the explicit stages, we can call out that just because something has moved to being an experimental feature does not mean it will become part of the project. Instead, there is a review that happens that allows these experimental features to land in the project.

This is all assuming a post-1.0 view for Nu, so that every step is very intentional. Getting to 1.0, being agile is going to be important, so like you mention, we're going to want to be flexible.

Reply:
That's good feedback. How about we lower the requirements for accepting an RFC, for now? That can still be adjusted later.

For example, we could do this:
```diff
- - Before actually entering FCP, *all* team members must sign off;
+ - Before actually entering FCP, *at least two* team members must sign off;
```

Reply:
@jzaefferer - that sounds good. We could even skip the FCP stage. If the proposal has gotten to the point where two team members sign off, it should be good enough to go ahead and implement.

Reply:
Makes sense! I've dropped the entire FCP stage.  This simplifies 'What the process is' quite a bit. I've replaced it with two simpler variations:

> - When the RFC is in a state that justifies community attention, it should be
    advertised widely, e.g. in [This Week in Nushell](https://www.notion.so/This-Week-in-Nu-ed589e19f1be47fa9ef562c1757e7ba1).
    This way all stakeholders have a chance to lodge any final objections
    before a decision is reached.
> - At some point after advertising the RFC was advertised publicly, a member
    of the team will propose a decision for the RFC (merge, close, or postpone).
    Another team member must sign off on that decision, and can then take the
    appropriate action. When closing or postponing, the reasons need to be well
    documented in the PR.

This still provides a certain minimal duration, but without the FCP overhead.